kolibri/core/assets/src/vue/tabs/index.vue

```diff
@@ -39,7 +39,7 @@
     },
     methods: {
       addTabListeners() {
-        const tabs = this.$refs.tabs.$el.querySelectorAll('.ui-tab-header-item');
+        const tabs = Array.from(this.$refs.tabs.$el.querySelectorAll('.ui-tab-header-item'));
```

Comment:
Alternatively, can do this: https://github.com/learningequality/kolibri-exercise-perseus-plugin/pull/33/files#diff-f249266c75182c1fa10f06fd5e7128eaR221

But this works too.